Packs/MailSenderNew/Integrations/MailSenderNew/MailSenderNew.yml

```
@@ -156,5 +166,5 @@ script:
   subtype: python2
   dockerimage: demisto/python:2.7.18.27799
 tests:
-- Mail Sender (New) Test
+- Mail Sender (New)
```

```suggestion
- Mail Sender (New) Test
```
Let's leave it like this:
1) This is indeed a test playbook
2) It fails the validation